apps/src/templates/rubrics/AiAssessmentFeedback.jsx

```diff
@@ -30,21 +31,34 @@ export default function AiAssessmentFeedback({learningGoalKey}) {
   const receivedTimer = useRef();
   const disappearAfter = 20000;
 
+  const baseUrl = '/learning_goal_ai_evaluation_feedbacks';
+
   const radioAiFeedbackCallback = radioButtonData => {
     setAIFeedback(radioButtonData);
+    if (radioButtonData === thumbsupval) {
+      const bodyData = JSON.stringify({
+        learningGoalAiEvaluationId: aiEvalInfo.id,
+        aiFeedbackApproval: thumbsupval,
+      });
+      HttpClient.post(`${baseUrl}`, bodyData, true, {
+        'Content-Type': 'application/json',
+      });
+    }
   };
 
   const submitAiFeedbackCallback = () => {
     const bodyData = JSON.stringify({
-      learningGoalKey: learningGoalKey,
-      aiFalseNeg: aiFalseNeg,
-      aiFalsePos: aiFalsePos,
-      aiVague: aiVague,
-      aiOther: aiFeedbackOther,
-      aiOtherContent: aiOtherContent,
+      learningGoalAiEvaluationId: aiEvalInfo.id,
+      aiFeedbackApproval: aiFeedback,
+      falsePositive: aiFalsePos,
+      falseNegative: aiFalseNeg,
+      Vague: aiVague,
```

Comment:
Please add a brief comment indicating that the capitalization here is to work around a bug. even if we don't fully understand it, it would be nice to tip ourselves off that lower-casing this key name will cause problems.